stdlib/atomic.mli

```diff
@@ -43,24 +43,23 @@ val make : 'a @ contended -> 'a t
     enhance performance.
 
     CR ocaml 5 all-runtime5: does not support runtime4 *)
-
-val make_contended : 'a @ contended -> 'a t
+val make_contended : 'a @ portable contended -> 'a t
 
 (** Get the current value of the atomic reference. *)
-val get_safe : 'a t -> 'a @ contended
+val get_safe : 'a t -> 'a @ portable contended
 
 (** Set a new value for the atomic reference. *)
-val set : 'a t -> 'a @ contended -> unit
+val set : 'a t -> 'a @ portable contended -> unit
```

Comment:
Our current goal is that our version of Stdlib is a valid implementation of the upstream Stdlib. Adding `portable` here makes `set` a weaker definition than in upstream's. We should add `set_safe` and alert `set`.